backend/question/migrations/0007_modeltranslation_populate_zh_hans.py

```diff
@@ -0,0 +1,29 @@
+from django.db import migrations
+from question.questions import create_default_questions
+from question import questions
+from django.core import management
+from django.utils import translation
+
+
+def populate_translation_fields(apps, schema_editor):
+
+    QuestionH = apps.get_model("question", "Question")
```

Comment:
Why call the model definitions `{}H`? I don't get what the `H` stands for, and in other migrations we just refer to this as the model name.